src/module/apps/itemImport/importer/VehicleModImporter.ts

```diff
@@ -0,0 +1,75 @@
+import { DataImporter } from './DataImporter';
+import { ImportHelper } from '../helper/ImportHelper';
+import { Constants } from './Constants';
+import { VehicleModParserBase } from '../parser/mod/VehicleModParserBase';
+import { UpdateActionFlow } from '../../../item/flows/UpdateActionFlow';
+
+export class VehicleModImporter extends DataImporter<Shadowrun.ModificationItemData, Shadowrun.ModificationData> {
+    public override categoryTranslations: any;
+    public accessoryTranslations: any;
+    public files = ['vehicles.xml'];
+
+    CanParse(jsonObject: object): boolean {
+        return jsonObject.hasOwnProperty('mods') && jsonObject['mods'].hasOwnProperty('mod');
+    }
+
+    ExtractTranslation() {
+        if (!DataImporter.jsoni18n) {
+            return;
+        }
+
+        let jsonWeaponsi18n = ImportHelper.ExtractDataFileTranslation(DataImporter.jsoni18n, this.files[0]);
```

Comment:
Use const for not re-assigned variables: jsonWeaponsi18n, datas, jsonDatas, jsonData, folder